opal/jquery_app_live_coding.rb

```diff
@@ -39,7 +39,7 @@
     37 => :left,
     39 => :right,
     38 => :up,
-    40 => :down,
+    40 => :down
```

Comment:
I like the comma on the last line, because it makes it easier to add another line